acceptance/lib/helper.rb

```diff
@@ -34,7 +34,7 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "7f71e72e936f460e136b25880c6792d9a20decdf",
+                         "8e8866b3da044f405a4e651cf374ab24a435767f",
```

Comment:
@adrienthebo I think this one needs to be the fully-qualified Puppet Agent SHA, not the Puppet submodule SHA.  fyi, I had already put this PR up earlier today with this change - see #1368.  If you want to fix this one up for this line, I can close mine out.